modules/Category/Category.tsx

```diff
@@ -1,9 +1,21 @@
 import { Category } from '@prezly/sdk/dist/types';
+import Link from 'next/link';
 
 type Props = {
     category: Category
 };
 
-const CategoryComponent = ({ category }: Props) => <span>{category.display_name}</span>;
+const CategoryComponent = ({ category }: Props) => {
+    // Use first available locale with a slug
+    // Change this for multilang support
+    const locales = Object.keys(category.i18n);
+    const locale = locales.filter((l) => !!category.i18n[l].slug)[0] || locales[0];
```

Comment:
```suggestion
    const locale = locales.find((locale) => category.i18n[locale].slug) || locales[0];
```

Reply:
that would shadow the locale on the left, and eslint complains about that